packages/theme/src/base/types.ts

```diff
@@ -82,6 +82,9 @@ export type BackstagePaletteAdditions = {
     closeButtonColor?: string;
     warning?: string;
   };
+  entityStarButton: {
```

Comment:
```suggestion
  starButton: {
```
I wonder if it's worth dropping `entity` from the name and just naming it starButton? I'm thinking that there might be cases where you want to use the same styling but for some other type of star that does not strictly relate to an entity.